mesonbuild/msubprojects.py

```diff
@@ -457,6 +460,13 @@ def checkout(self) -> bool:
 
     def download(self) -> bool:
         self.log(f'Download {self.wrap.name}...')
+        options = T.cast('DownloadArguments', self.options)
```

Comment:
Can you add `from __future__ import annotations` to remove the quotes?

Reply:
No. This isn't an annotation.

Reply:
(Also the file already uses that future import, because all files have it.)